### PR TITLE
Welcome new programmer

### PR DESCRIPTION
**Pull request by SzilvasiPeter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # hello-world
 This is my first repository.
 
-This is my first commit.
\ No newline at end of file
+This is my first commit.
+
+Welcome to Github Sleepy!
```
Comment (savolyialmos) [inline README.md]: Please change the "Sleepy" word to "Álmos"!

**Comment by SzilvasiPeter** [comment]
Hello!

File: README.md (hunk 1 above)
Comment (savolyialmos) [reply README.md:6]: Please change the "Sleepy" word to "Álmos"!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # hello-world
```
Comment (savolyialmos) [inline README.md:1]: Comment

Comment (SzilvasiPeter) [reply README.md:1]: Comment 2

File: README.md (hunk 1 above)
Comment (SzilvasiPeter) [reply README.md]: @savolyialmos Done by 74c5437dc99e8129ef3e8b9dff30462a52a3239b commit.

**Pull request merged**